Sources/MapboxCoreNavigation/EHorizon/RoadObjectType.swift

```diff
@@ -2,31 +2,60 @@ import Foundation
 import MapboxDirections
 import MapboxNavigationNative
 
-/** Type of the road object */
+/**
+ Type of the road object.
+ */
 public enum RoadObjectType {
 
-    /** Road object represents some road incident */
+    /**
+     An alert providing information about incidents on a route. Incidents can include *congestion*,
+     *massTransit*, and more (see `Kind` for the full list of incident types).
```

Comment:
Technically, we could use backticks to link to the individual cases of `Incident.Kind`, but I think it’s sufficient to link to `Incident.Kind` and refer to these example kinds in plain English.

```suggestion
     An alert providing information about incidents on a route. Incidents can include congestion,
     mass Transit, and more (see `Incident.Kind` for the full list of incident types).
```